### PR TITLE
feat(promo-banner): introducing new pattern (behind feature flag)

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Related Ticket(s)

React: Promo banner (behind feature flag) #3928

### Description

This section first has appeared on the new [static Support page L0](https://www.ibm.com/testing/support/). Check out the [visual specs here](https://app.abstract.com/projects/7f83c114-7627-4169-8af7-6a85e7147ea6/branches/master/commits/latest/files/01ee3bd8-c63d-4b3b-bddb-aabaddb5c31d/layers/88A2BB62-9234-4F68-B238-BBA28C43B876?collectionId=c963cb53-73c2-42a5-af39-f5538e24c454&collectionLayerId=dd10dfb7-71be-4c98-9562-73c151d3236b).

### Changelog

**New**

- `PromoBanner` pattern;

### CODE REVIEW

File: packages/react/src/components/PromoBanner/PromoBanner.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import LinkWithIcon from '../LinkWithIcon/LinkWithIcon';
+import markdownToHtml from '@carbon/ibmdotcom-utilities/es/utilities/markdownToHtml/markdownToHtml';
+import PropTypes from 'prop-types';
+import React from 'react';
+import settings from 'carbon-components/es/globals/js/settings';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+const PromoBanner = ({ banner, copy, cta }) =>
```
Comment (jeffchew) [inline packages/react/src/components/PromoBanner/PromoBanner.js:18]: @guilhermelMoraes can you use the `featureFlag` utility as a wrapper to this? Should be similar to how we created `CalloutData` (as an example).

File: packages/react/src/components/PromoBanner/README.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  const copy =
+    'Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin.';
+  const cta = {
+    icon: <ArrowRight20 />
+    copy: 'External link text',
+    href: 'https://example.com',
+  };
+
+  return (
+    <ContentItemHorizontal
```
Comment (jeffchew) [inline packages/react/src/components/PromoBanner/README.stories.mdx:63]: Shouldn't this be `PromoBanner`?

File: packages/react/src/components/PromoBanner/README.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      banner={banner}
+      copy={copy}
+      cta={cta}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
```
Comment (jeffchew) [inline packages/react/src/components/PromoBanner/README.stories.mdx:73]: Can you add documentation on adding the feature flag `DDS_PROMO_BANNER` to use this functionality?

File: packages/react/src/components/PromoBanner/README.stories.mdx (hunk 2 above)
Comment (guilhermelMoraes) [reply packages/react/src/components/PromoBanner/README.stories.mdx:63]: Ooops!

**Review by jeffchew — APPROVED** [review]
Great add, thank you @guilhermelMoraes !

**Pull request merged**